tap_xenon.tex

```diff
@@ -0,0 +1,418 @@
+online fuel reprocessing system is inactive. Another feature of the \gls{MSR}, 
+its circulating liquid fuel and corresponding delayed neutron precursor drift, 
+is also not treated here.
+
+Much of the analysis herein used a full-core 3-D model of the \gls{TAP}  
+developed using the continuous-energy Serpent 2 Monte Carlo reactor physics 
+software. The \gls{PWR} transient analysis has been done for a single-assembly 
+model with burnable poison (gadolinium) provided with Serpent 
+\cite{leppanen_serpent_2015}. All calculations presented in this paper were 
+performed using the Serpent 2 version 2.1.31 with JEFF-3.1.2 nuclear data 
+\cite{oecd/nea_jeff-3.1.2_2014}.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Description of the actual work}
+\subsection{Transatomic Power Reactor Design}
+The \gls{TAP} \gls{MSR} concept is very similar to original \gls{MSRE} design 
+developed 
+by \gls{ORNL} \cite{haubenreich_experience_1970} but has two major  
+innovations: 
+the fuel salt composition and the moderator. The \gls{MSRE}'s 
+LiF-BeF$_2$-ZrF$_4$-UF$_4$ salt has been substituted with LiF-UF$_4$ salt 
+which 
+allows for an increase in the uranium concentration within the fuel salt from 
+0.9 to 
+27.5\% while maintaining a relatively low melting point (490$^{\circ}$C 
+compared 
+with 434$^{\circ}$C for the original \gls{MSRE}'s salt) 
+\cite{betzler_assessment_2017}. The graphite has a very high 
+thermal scattering cross section which makes it a perfect moderator but has 
+a few major drawbacks: 
+(1) the low lethargy gain per collision requires a large volume of moderator 
+to be present to reach criticality, which leads to a larger core and obstructs 
+the core power density; (2) even special 
+reactor-grade graphite has relatively high porosity, consequently, it holds
+gaseous \glspl{FP} 
+(e.g., tritium, xenon) in pores; (3) the reactor graphite lifespan in a 
+commercial 
+reactor is about 10 years \cite{robertson_conceptual_1971}. To resolve these 
+issues, the \gls{TAP} concept uses another 
+moderator, namely, zirconium hydride (ZrH$_{1.66}$), allowing for a more 
+compact core and a 
+significant increase in power density. These two innovative design choices,  
+together with a configurable moderator 
+(the moderator-to-fuel ratio can be changed during regular maintenance 
+shutdown), 
+facilitate the commercial deployment of this conceptual design in the current 
+commercially available 5\% \gls{LEU} fuel cycle. 
+
+The \gls{TAP} \gls{MSR} primary loop contains the reactor core volume 
+(including the zirconium hydride moderator rods with silicone carbide 
+cladding), pumps, and primary heat exchanger. Pumps circulate the 
+LiF-(Act)F$_4$ fuel salt through the primary loop. The pumps, vessels, tanks, 
+and piping are made of a nickel-based alloy (similar to Hastelloy-N\footnote{ 
+Hastelloy-N  is very common in reactors now but have been studied and 
```

Comment:
do you mean not very common?

Reply:
Why? I think it is very common. Its commercial product now and being used widely in the energy generation industry.

Reply:
The word `but` in the sentence made me think the metal was not commonly used. Maybe change `but have` to `and has`

Reply:
Gotcha